recipes/selestim/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "1.1.7" %}
+{% set name = "selestim" %}
+
+package:
+  name: {{ name }}
+  version: '{{ version }}'
+source:
+  fn: 'SelEstim_1.1.7.zip'
+  url: http://www1.montpellier.inra.fr/CBGP/software/selestim/files/SelEstim_1.1.7.zip
+  sha256: a2a8a8d37a9adf4b6637ef0cac597a48296a83c7d50e62a1dd31fdc385510f21
+build:
+  number: 0
+requirements:
+  build:
+    - gcc
+    - openmp
```

Comment:
please pin the openmp version, `- openmp 4.0.*  # [osx]`
Same for the run-time dependencies.